src/registry_schemas/schemas/ppr/searchSummary.json

```diff
@@ -29,7 +29,7 @@
                     "type": "string",
                     "maxLength": 2,
                     "enum": ["SA", "RL", "FR", "LT", "MH", "SG", "FL", "FA", "FS", "MR", "CC", "CT", "DP", "ET", "FO", "FT", "HR", "IP", "LO", "MI", "OT"
-                        , "PG", "PS", "IT", "RA", "SS", "TL", "HN", "ML", "MN", "PN", "WL", "TF", "TA", "TG", "TM"],
+                        , "PG", "PS", "IT", "RA", "SS", "TL", "HN", "ML", "MN", "PN", "WL", "TF", "TA", "TG", "TM", "MD", "PT", "SC", "TO", "SV"],
```

Comment:
What do these Financial Statement Codes correlate too?

Reply:
They are the set of PPR registration types. The base registration is a Financing Statement.

Reply:
copy that, thank you!